src/generic_ui/polymer/root.html

```diff
@@ -272,7 +272,7 @@
           </div>
         </core-toolbar>
 
-        <uproxy-bubble on-closed='{{ closedWelcome }}' active='{{ !model.globalSettings.hasSeenWelcome && !statsDialogOrBubbleOpen }}'>
+        <uproxy-bubble class='core-header' on-closed='{{ closedWelcome }}' active='{{ !model.globalSettings.hasSeenWelcome && !statsDialogOrBubbleOpen }}'>
```

Comment:
Can you just add a comment here explaining this?  We don't ourselves set any style for .core-header, I assume that instead it's the Polymer element we include which is setting the style.  Thanks